index.html

```diff
@@ -4,10 +4,95 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link href="./style.css" rel="stylesheet">
     <title>Document</title>
 </head>
 
 <body>
+    <header>
+        <div class="navbar">
+            <nav>
+                <img class="logo" src="./img/Takeoff.png">
+                <div class="menu">
+                    <p><a class="tag-menu" href="#">OVERVIEW</a></p>
+                    <p><a class="tag-menu" href="#">HOW IT WORKS</a></p>
+                    <p><a class="tag-menu" href="#">FEATURES</a></p>
+                    <p><a class="tag-menu" href="#">APP</a></p>
+                    <p><a class="tag-menu" href="#">CONTACT</a></p>
+                </div>
+                <div class="btns-nav">
+                    <button id="btn-login">SING IN</button>
```

Comment:
It's wrong to use id for button styling. Use class instead.

Reply:
That's fixed!
HTML:
<div class="btns-nav">
                    <button class="btn-nav">SING IN</button>
                    <span>
                        <button class="btn-nav">CONTACT</button>
                    </span>
                </div>
CSS:
.btns-nav>span>button {
        margin-left: 12px;
        background-color: #094b58;
    }